src/Symfony/Bridge/Twig/Tests/Extension/HttpKernelExtensionTest.php

```diff
@@ -46,7 +46,13 @@ public function testUnknownFragmentRenderer()
         ;
         $renderer = new FragmentHandler(array(), false, $context);
 
-        $this->setExpectedException('InvalidArgumentException', 'The "inline" renderer does not exist.');
+        if (method_exists($this, 'expectException')) {
+            $this->expectException('InvalidArgumentException');
```

Comment:
Possible to use annotation instead here? That's the last thing remaining, thanks for the work.